daml-assistant/daml-helper/src/DA/Daml/Helper/Main.hs

```diff
@@ -32,71 +33,132 @@ data Command
     | Init { targetFolderM :: Maybe FilePath }
     | ListTemplates
     | Start { sandboxPortM :: Maybe SandboxPort, openBrowser :: OpenBrowser, startNavigator :: StartNavigator, onStartM :: Maybe String, waitForSignal :: WaitForSignal }
-    | Deploy        { flags :: HostAndPortFlags }
-    | ListParties   { flags :: HostAndPortFlags }
-    | AllocateParty { flags :: HostAndPortFlags , party :: String }
+    | Deploy { flags :: HostAndPortFlags }
+    | LedgerListParties { flags :: HostAndPortFlags }
```

Comment:
Maybe we should introduce a level of nesting here to match the CLI structure, i.e., introduce a new `LedgerCommand` type and then only have a single constructor with a field of type `LedgerCommand` in `Command`.

Reply:
👍 I'll do this in another PR